ports/esp32/machine_uart.c

```diff
@@ -414,13 +416,15 @@ STATIC mp_obj_t machine_uart_sendbreak(mp_obj_t self_in) {
         // Cannot synthesise break
         mp_raise_OSError(MP_EPERM);
     }
```

Comment:
I think this set of if-statements can now be removed, it should be enough to just decrease the baudrate.

Reply:
That's true.

Reply:
Updated.

Reply:
Looks good!